### ISSUE TITLE
[grid] Selecting rows using the keyboard

### ISSUE THREAD

**Issue by jpzwarte** [sl-design-system/components#2023]
### Provide a general summary of the issue here

This is a followup of #1526 

https://storybook.sanomalearning.design/?path=/story/grid-selection--clickable-row

At the moment, if a grid has no interactive elements (such as checkbox, button etc.), you cannot TAB to a row. You can still (in the story above) click on a row to select it. 

It should be possible to select the row using the keyboard as well.

### 👤 Your name

Jeroen

### 🧢 Your Product/Team

SLDS
